migaloo/assetlist.json

```diff
@@ -22,6 +22,69 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/whitewhale.png"
         },
         "coingecko_id": ""
+      },
+      {
+      "description": "ampWHALE",
+      "denom_units": [
+        {
+          "denom": "factory/migaloo1436kxs0w2es6xlqpp9rd35e3d0cjnw4sv8j3a7483sgks29jqwgshqdky4/ampWHALE",
+          "exponent": 0
+        },
+        {
+          "denom": "ampWHALE",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/migaloo1436kxs0w2es6xlqpp9rd35e3d0cjnw4sv8j3a7483sgks29jqwgshqdky4/ampWHALE",
+      "name": "ampWHALE",
+      "display": "ampWHALE",
+      "symbol": "ampWHALE",
+      "coingecko_id": "ampWHALE",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/ampWhale.svg"
+      }
+    },
+    {
+      "description": "boneWHALE",
+      "denom_units": [
+        {
+          "denom": "factory/migaloo1mf6ptkssddfmxvhdx0ech0k03ktp6kf9yk59renau2gvht3nq2gqdhts4u/boneWhale",
+          "exponent": 0
+        },
+        {
+          "denom": "boneWHALE",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/migaloo1mf6ptkssddfmxvhdx0ech0k03ktp6kf9yk59renau2gvht3nq2gqdhts4u/boneWhale",
+      "name": "boneWHALE",
+      "display": "bWHALE",
+      "symbol": "bWHALE",
+      "coingecko_id": "bWHALE",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/bWHALE.svg"
+      }
+    },
+    {
+      "description": "FABLE",
+      "denom_units": [
+        {
+          "denom": "factory/migaloo18a9m9stu3dyvewwcq9qmp85euxqcvln5mefync/fable",
+          "exponent": 0
+        },
+        {
+          "denom": "fable",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/migaloo18a9m9stu3dyvewwcq9qmp85euxqcvln5mefync/fable",
+      "name": "FABLE",
+      "display": "FABLE",
```

Comment:
the display [denom] must match one of the denom_units defined above. so this should be 'fable' (lowercase).